Changes to OASIS templates and Javascript

ndjc opened the pull request:
A set of changes to OASIS templates, scripts, and CSS, to generate the right citation block, and to fix a couple of other small layout issues.

Commit:
adding stopgap jquery map to get dev env working

Commit:
Reverting stopgap jquery try

sspeiche committed:
Merge branch 'gh-pages' of https://github.com/sspeiche/respec into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
merge

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
merge

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge of feature/oasis-style

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
fix n3 release problem

sspeiche committed:
merge from oasis-style

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

sspeiche committed:
Merge branch 'feature/oasis-style' into gh-pages

ndjc committed:
Merge branch 'feature/oasis-style' into gh-pages
Sort shape tables by property name

sspeiche committed:
Merge branch 'gh-pages' of https://github.com/sspeiche/respec into feature/oasis-style

ndjc committed:
Merge branch 'feature/oasis-style' into gh-pages

ndjc committed:
Merge branch 'feature/oasis-style' into gh-pages:
Sort classes, properties, and individuals by name in OASIS vocabularies

ndjc committed:
Merge branch 'gh-pages' of https://github.com/ndjc/respec into gh-pages

ndjc committed:
Ignore .md.html file

ndjc committed:
Ignore .md.html files

ndjc committed:
Various fixes for OASIS format specs

  Support for OASIS citation section
  Fix ordering of sotd generation
  Update copyright generation

ndjc committed:
Merge branch 'feature/oasis-style' into gh-pages

Commit:
Make oslc:range property multiValue

* When we want to limit the range of a property in more than two types,
  we have to specify its oslc:range as them respectively.
* This also fixes when multi-valued property is flagged, ReSpec wouldn't
  get their neither object nor literal.

* see also
  `https://lists.oasis-open.org/archives/oslc-promcode/201505/msg00067.html`

Commit:
Merge remote-tracking branch 'origin/gh-pages' into feature/oasis-style

ndjc committed:
Allow ED docs to have no previous version

ndjc committed:
Merge branch 'feature/oasis-style' into gh-pages

ndjc committed:
Merge pull request #2 from imunolion/gh-pages

gh-pages update for 0.0.8

ndjc committed:
Merge remote-tracking branch 'sspeiche/feature/oasis-style' into feature/oasis-style

ndjc committed:
Merge changes from Steve

ndjc committed:
Resolve conflicts and merge

ndjc committed:
Merge branch 'gh-pages' into feature/oasis-style

ndjc committed:
Use ndjc github repo as the master (at least for testing purposes)

ndjc committed:
Build version with ndjc github as the master, and with chamges from imunolion

ndjc committed:
v0.0.9

ndjc committed:
v0.0.9

ndjc committed:
v0.0.9

ndjc committed:
v0.0.9

ndjc committed:
v0.0.9

ndjc committed:
v0.0.9

ndjc committed:
v0.0.9

ndjc committed:
v0.0.9

ndjc committed:
v0.0.9

ndjc committed:
v0.0.9

ndjc committed:
Add oslc:representation to shape template

ndjc committed:
Update shape template to handle oslc:representation as array

ndjc committed:
Update shape.js to pass oslc:representation as array
and to substitute namespace prefixes correctly

ndjc committed:
Add special handler for oslc:range

ndjc committed:
Bump version number

ndjc committed:
v0.1.0

ndjc committed:
v0.1.0

sspeiche commented on js/oasis/style.js:
```diff
@@ -13,7 +13,7 @@ define(
             	var css = "https://www.oasis-open.org/spectools/css/spec.css";
                 utils.linkCSS(doc, css);
-                utils.linkCSS(doc, 'https://sspeiche.github.io/respec/js/oasis/css/default.css');
+                utils.linkCSS(doc, 'https://ndjc.github.io/respec/js/oasis/css/default.css');
```
Do you really plan to cross host stuff between my both yours and mine?

sspeiche commented:
I see this PR has a couple references to ndjc.github.io, if I accept this PR as-is then what I host at sspeiche.github.io will reference ndjc.github.io. If you want to switch the "primary" repo to be ndjc/repec for the oasis work it would be good to know and perhaps I could change my repo to just be a fork of yours and yours a fork of w3c/respec.

ndjc commented:
I did not mean to include the changes to use ndjc.github.io in the pull request.  I have made those changes in many places in my own branch now, because I needed to make a whole bunch of additional changes as we get ready for public review - and I also added some changes for the PROMCODE TC. At this point, given the scale of some of the changes I have made after this particular pull request, it may be better to do as you suggest. I have already suggested to the Core TC that we use my repo as the temporary source for ReSpec until/unless we can get OASIS to host or sponsor one - which they are still considering.

sspeiche commented:
I believe I could give you access to me repo as a "committer" as well, then
you can merge pull requests, etc as a stopgap until OASIS does something
more official.

Or I could just be quicker about merging in your changes, you just hit me
on a bad 3 week travel cycle

ndjc commented:
I'd be OK either way - we plan to discuss it at this week's OASIS OSLC meeting. The next merge from my repo to yours would be quite large in terms of the number of commits, partly because I made a mistake in merging a pull request from the PROMCODE team into my repo - I pulled it into gh-pages, and then had to merge gh-pages into feature/oasis-style rather than vice versa. That didn't make any difference to the content that I have, but it did add in a large number of commit numbers!

sspeiche commented:
You can squash all the commits down to a smaller set. It is a common best practice to make the commit history clean and have near 1 commit per PR.  I'd suggest updating this PR to remove the ndjc references and squashing.  See https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

Commit:
Update location of specref biblio

Commit:
Change bibliography to use W3 site
Set representation to N/A for literals

Commit:
Change bibliography to use W3 site
Set representation to N/A for literals

Commit:
Change bibliography to use W3 site
Set representation to N/A for literals

Commit:
v0.1.2

Commit:
v0.1.3

Commit:
v0.1.4

Commit:
v0.1.5

Commit:
v0.1.6

Commit:
v0.1.7

Commit:
v0.1.8

Commit:
Add citation label style

Commit:
v0.2.0

Commit:
Add OASIS version of structure.js and biblio.js

Commit:
v0.2.1

Commit:
Better handling of related work and supersedes
Remove support for oslcVersion - that should now be embedded in the
title

Commit:
v0.2.2

Commit:
v0.2.3

Commit:
v0.2.4

Commit:
v0.2.5

Commit:
Update example shapes and vocab

Commit:
Update examples & template spec

Commit:
v0.2.6

Commit:
v0.2.7

ndjc committed:
v0.2.9

ndjc committed:
v0.3.0

ndjc committed:
v0.3.1

ndjc committed:
v0.3.2

ndjc committed:
v0.3.3

ndjc committed:
Add 'v' in front of version number in title,
and change TC URI from http to https

ndjc committed:
Titles of references/related docs are in italic,
TC URI uses https,
and order of version references changed

ndjc committed:
Use https for TC and OASIS URIs

ndjc committed:
Add paragraph about machine-readable content

ndjc committed:
v0.3.4

Commit:
v0.3.5
Changed , to . between title and URL in versions.
Added (Authoritative) after this and latest version
Added traling / to wgURI in Status
Added new Status text as per OASIS.

ndjc committed:
v0.3.5

ndjc committed:
v0.3.6

ndjc committed:
Remove some obsolete logging messages

ndjc committed:
v0.3.6

ndjc committed:
Remove debug messages

ndjc committed:
v0.3.6

ndjc committed:
v0.3.6

ndjc committed:
v0.3.7

Commit:
Updates for OASIS document guidelines and to generate more directly publishable documents

Add the .pdf links to This version, Previous version and Latest version
Changed a , to a . in the citation title as requested by Chet
In the Status section, changed "Committee Specification Public Review Draft" to "specification" as requested by Chet to make the section document state independent
updated the package.json dependencies and built with Node.js v11.8.1
minor change to the README.md

Commit:
v0.3.8

Includes my previous updates

ndjc committed:
Restore correct contents of build 0.3.7

ndjc committed:
Update README.md to apply more specifically to OASIS

ndjc committed:
Remove obsolete W3C builds of ReSpec from OASIS fork

ndjc committed:
v0.3.9

ndjc committed:
v0.3.10

ndjc committed:
v1.0.0

ndjc committed:
v1.0.1

Commit:
Mark this fork as obsolete

Commit:
Mark as obsolete
